static/src/javascripts/projects/journalism/views/campaignForm.html

```diff
@@ -40,14 +40,13 @@ <h4 class="campaign--snippet__headline"> <%= data.title %></h4>
                 <label for="<%=field.name%>"
                        class="form_field_label">
                     <%= field.label %>
-                    <span><%= field.description %></span>
+                    <% if(field.description) { %>
+                        <span>(<%= field.description %>)</span>
+                    <% } %>
                 </label>
 
                 <% if(field.type === "textarea") { %>
-                <textarea name="field_<%=field.id%>"
-                <% if (field.required == "1") { %> required <% } %>
-                />
-                </textarea>
+                <textarea name="field_<%=field.id%>"<% if (field.required == "1") { %> required <% } %>/></textarea>
```

Comment:
Well done removing the extra whitespace 👍

Reply:
@regiskuckaertz Please review the change.